katas/content/complex_arithmetic/complex_addition/index.md

```diff
@@ -0,0 +1,28 @@
+**Inputs:**
+
+1. A complex number $x = a + bi$.
+2. A complex number $y = c + di$.
+
+**Goal:**
+Return the sum of x and y. That is $x + y = z = g + hi$.
+
+Revise the following Q# code to return the sum of x and y.
+
+<details>
+  <summary><b>Need a hint?</b></summary>
+  
+* Use the Complex data type defined in the Q# math library. For example, $x = a + bi$:
```

Comment:
Introducing Complex data type should not be part of the hint, they'll need this information no matter what, it's better to make it part of the Complex Numbers lesson